compiler/qsc_qasm/src/ast_builder.rs

```diff
@@ -1328,6 +1350,11 @@ pub(crate) fn map_qsharp_type_to_ast_ty(output_ty: &crate::types::Type) -> Ty {
         crate::types::Type::IntArray(dims, _) => build_array_type_name("Int", dims),
         crate::types::Type::DoubleArray(dims) => build_array_type_name("Double", dims),
         crate::types::Type::BoolArray(dims, _) => build_array_type_name("Bool", dims),
+        crate::types::Type::ComplexArray(dims, _) => {
+            let ty = build_complex_ty_ident();
+            wrap_array_ty_by_dims(dims, ty)
+        }
+        crate::types::Type::AngleArray(dims, _) => build_array_type_name("__Angle__", dims),
```

Comment:
This name changed a few days ago. Any code using this should be erroring. It there isn't an error then there is untested code.